### PR TITLE
Update README and evaluate_onmt_predictions.py

### PR DESCRIPTION
**Pull request by hao-fang**
1. Add links to `SMCalFlow 1.0` since now they are removed on the leaderboard webpage.
2. Report accuracy ignoring `refer_are_correct`, which is now also reported on the leaderboard.

### CODE REVIEW

File: src/dataflow/onmt_helpers/create_onmt_prediction_report.py [hunk 1]
```diff
@@ -81,6 +85,14 @@ def is_correct_leaderboard(self) -> bool:
             and self.program_execution_oracle.refer_are_correct
         )
 
+    @property
+    def is_correct_leaderboard_ignoring_refer(self) -> bool:
+        """Returns true if the gold and the prediction match after canonicalization, ignoring the `refer_are_correct`.
+
+        Use this metric is you only care about the accuracy of the semantic parser, ignoring the execution of `refer`.
+        """
+        return self.gold_canonical == self.prediction_canonical
```
Comment (adampauls) [inline src/dataflow/onmt_helpers/create_onmt_prediction_report.py:94]: :sheep: can you make line 84 be is_correct_leaderboard_ignoring_refer(self) and self.program_execution_oracle.refer_are_correct

**Review by adampauls — APPROVED** [review]
Thanks!

**Review by sammthomson — APPROVED** [review]
👍

File: src/dataflow/onmt_helpers/create_onmt_prediction_report.py [hunk 2]
```diff
@@ -99,7 +111,9 @@ def flatten(self) -> Dict[str, Union[str, int]]:
                 "predictionCanonical": self.prediction_canonical,
                 "oracleResolveAreCorrect": self.program_execution_oracle.refer_are_correct,
                 "isCorrect": self.is_correct,
+                "isCorrectIgnoringRefer": self.is_correct_ignoring_refer,
```
[the remark this answers is not in the document]
Comment (hao-fang) [reply src/dataflow/onmt_helpers/create_onmt_prediction_report.py:114]: I think we need it for reproducing the TACL2020 paper results.

**Pull request merged**